frontend/src/renderer/components/Process/Tabs/BatchProcessingRoles.svelte

```diff
@@ -3,7 +3,7 @@
     import { processState, batchProcessState, batchImagesA, batchImagesB } from "@util/stores";
     import Dropbox from "@components/Process/Dropbox.svelte";
     import {get, isEmpty, each, includes} from "lodash";
-    import { autoSortImages } from "@util/autoSortStandards.svelte";
+    import { autoSortBatchImages } from "@util/autoSortStandards.svelte";
```

Comment:
Do you think you should import both autoSortImages and autoSortBatch Images and then just have a flag to determine which sort to use? Not sure how vastly different they are functionality wise

Reply:
also don't forget to merge in beyond_rgb_2.0 before completing

Reply:
The batch images page and the single image page are entirely different pages and it conditionally decides which page to render, so the single image page still uses the old auto sort images function. I don't think there's ever a case where we would want to use the old autosort on the batch images page.